src/elements/ngm-drop-files.js

```diff
@@ -0,0 +1,66 @@
+import {html, css} from 'lit-element';
+
+import i18next from 'i18next';
+import {LitElementI18n} from '../i18n.js';
+
+const cancel = event => event.preventDefault();
+
+class NgmDropFiles extends LitElementI18n {
+
+  static get properties() {
+    return {
+      target: {type: String},
+    };
+  }
+
+  static get styles() {
+    return css`
+
+    `;
+  }
+
+  firstUpdated() {
+    this.dimmer = this.querySelector('.ui.dimmer');
```

Comment:
Can register listeners in the `connectedCallback` and unregister them on `disconnectedCallback`?

Reply:
fixed